15_3_tabs.js

```diff
@@ -0,0 +1,19 @@
+const buttons = document.querySelectorAll('button');
+const arrayTab = [...document.querySelectorAll('tab-panel div')];
+
+const divTabs = arrayTab.map(div => div.getAttribute('data-tabname'));
+
+buttons.forEach(button => {
+    button.addEventListener('click', function() {
+        const currentTab = document.querySelector('.show');
```

Comment:
Maybe try to segregate expressions inside that event listener using empty lines as we discussed today :)